### PR TITLE
chore: write tests for theme utils

### PR DESCRIPTION
**Pull request by EmiBemi**

### CODE REVIEW

File: packages/theme/src/lib/utils.tsx [hunk 1]
```diff
@@ -162,15 +162,15 @@ const hexToRgb = (hex: string) => {
       };
 };
 
-const luminance = (r: number, g: number, b: number) => {
-  var a = [r, g, b].map(function(v) {
+export const luminance = (r: number, g: number, b: number) => {
+  var a = [r, g, b].map(function (v) {
```
Comment (jensborch) [inline packages/theme/src/lib/utils.tsx:166]: Can't we use "let" her?

**Review by jensborch — CHANGES_REQUESTED** [review]

File: packages/base/src/lib/utils/strings.tsx [hunk 2]
```diff
@@ -1,6 +1,8 @@
 export const toCamel = (str: string) =>
-  str.replace(/(-[a-z])/g, $1 => $1.toUpperCase().replace('-', ''));
+  str.replace(/(-[a-z])/g, ($1) => $1.toUpperCase().replace('-', ''));
 
 export const toDashCase = (str: string) => {
-  return str.replace(/([A-Z])/g, $1 => '-' + $1.toLowerCase());
+  return str
+    .replace(/(^[a-z]+|[A-Z][a-z]*|[0-9]+)/g, ($1) => '-' + $1.toLowerCase())
```
Comment (jensborch) [inline packages/base/src/lib/utils/strings.tsx:6]: We can use `\d` instead of `[0-9]`

**Review by jensborch — CHANGES_REQUESTED** [review]

File: packages/base/src/lib/utils/strings.tsx (hunk 2 above)
Comment (EmiBemi) [reply packages/base/src/lib/utils/strings.tsx:6]: That works too.

File: packages/theme/src/lib/utils.tsx (hunk 1 above)
Comment (EmiBemi) [reply packages/theme/src/lib/utils.tsx:166]: Sure

**Pull request merged**